config/initializers/devise.rb

```diff
@@ -253,4 +253,5 @@
   # When using omniauth, Devise cannot automatically set Omniauth path,
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
+  config.omniauth :facebook, ENV['FACEBOOK_APP_ID'], ENV['FACEBOOK_SECRET']
```

Comment:
Would be better if we had these as `ENV.fetch('FACEBOOK_APP_ID')` and `ENV.fetch('FACEBOOK_SECRET')` – will error out if we haven't set them in production or locally but easier to debug.